core/src/interfacespec.cpp

```diff
@@ -25,6 +25,16 @@
 using namespace forte::literals;
 
 namespace forte {
+
+  namespace {
+    StringId getPortNameId(TPortId paPortId, std::span<const StringId> paPortNames) {
+      if (paPortId > paPortNames.size()) {
+        return ""_STRID;
```

Comment:
Since an empty string is a valid port name (i.e., function return variable), you may want to use an invalid string id here:

```suggestion
        return {};
```